changeuser.php

```diff
@@ -0,0 +1,184 @@
+<?php
```

Comment:
Bitte orientiere dich bei der Strukturierung deines codes am vorhanden Code. Diese Datei gehört z.b. in das api/ Verzeichnis. Verwende bitte auch die vorhandenen Funktionen wie apiCheckParams/apiCheck/apiAction zur Fehlerbehandlung und zur Strukturierung. Als Beispiel kannst du dir z.B. https://github.com/MackieLoeffel/weltfairsteher/blob/master/api/addTeacher.php anschauen.